src/PrimeBooking.Domain/Informative/Common/Error.cs

```diff
@@ -0,0 +1,3 @@
+namespace PrimeBooking.Domain.Informative.Common;
+
+public record Error(string ErrorCode, string ErrorType, string Detail, string? StackTrace);
```

Comment:
Do we need to add some enum to map it to proper status code? like enum StatusCode { NotFound, BadRequest, UnprocessableEntity etc. }

Reply:
Good idea, I will add the default one (HttpStatusCode)